README.md

```diff
@@ -17,4 +17,4 @@ To transform this skeleton in your plugin your may want to:
 * Update this `README.md`
 * Publish your plugin to the Hansken community!
 
-The Hansken extraction plugin developer guide contains further information on how to get started.
\ No newline at end of file
+The Hansken extraction plugin developer guide contains further information on how to get started.
```

Comment:
A casual Github passer-by might not have this guide. Can we be more specific where it is, or how you can get it?

Reply:
are we allowed to link to eminjenv here? 😅

Reply:
Not sure, maybe this guide is something to include here as well.